src/Server/Program.fs

```diff
@@ -42,7 +65,16 @@ let main args =
             |> Option.defaultValue Database.DatabaseType.FileSystem
 
         let port = getPortsOrDefault 8085us
-        WebServer.start database clientPath port
+        
+        WebHost
+            .CreateDefaultBuilder()
+            .UseWebRoot(clientPath)
+            .UseContentRoot(clientPath)
```

Comment:
I forgot about this. I think ContentRoot should be different from WebRoot (= clientPath). In our case we're using the repo root. Although I haven't seen so far any use of ContentRoot yet, as we save the root in another value for file system access, etc. I guess it's used by ASP.NET services.

Reply:
@alfonsogarciacaro no idea really :) I doubt because it's used in this such an ubiquitous `CreateDefaultBuilder` method.